src/zzz_od/auto_battle/auto_battle_dodge_context.py

```
@@ -87,7 +87,16 @@ def _record_loop(self) -> None:
         warnings.filterwarnings('ignore', category=SoundcardRuntimeWarning)
 
         try:
-            _mic = sc.get_microphone(id=str(sc.default_speaker().name), include_loopback=True)
+            # 按 WASAPI id 精确匹配默认扬声器的 loopback 设备（录制系统输出声音）。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)